### PR TITLE
Change appcode in appcat.yaml

### PR DESCRIPTION
**Pull request by slee1016**
Change appcode to nakadi-- AppCat error: an app cannot have the same appCode as sysCode.

SRE-4270

### CODE REVIEW

File: appcat.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 --- !<Application>
 id: "077a600f-29ce-443a-92b4-6d8656ae11ff"
 modelType: "Application"
-appCode: "eventhub"
```
Comment (manusha-pieris) [inline appcat.yaml:4]: @slee1016  Previously, in the nakadi POC time, we used the appCode as nakadi for this repo. Then after POC ended and started eventhub we changed the appCode from nakadi to eventhub. So shall we revise the appCode as event-hub now.

Comment (slee1016) [reply appcat.yaml:4]: Ok @manusha-pieris , I just changed the appCode to event-hub if you can confirm the change looks good

Comment (slee1016) [reply appcat.yaml:4]: Nvm just spoke to @jojijohn -- the appCode will be nakadi

**Review by manusha-pieris — APPROVED** [review]

**Pull request merged**